### DISCUSSION TITLE
Drawing new data in lightweight chart without refresh

### DISCUSSION THREAD

**Discussion by 0xdev070** [tradingview/lightweight-charts#864] [General]
How to add new data without re-render the chart?

**Comment by timocov** [comment]
> add new data without re-render the chart?

Could you elaborate your question please? What do you mean? Do you want to add a data but don't render it? Or what?

**Reply by 0xdev070** [reply]
I just want to display new data without re-render the graph.
I mean real time data
Is it available in lightweight chart?

**Reply by timocov** [reply]
> without re-render the graph.

how can you display new data without re-render? what do you mean under "re-render"?

**Comment by chung362** [comment]
Official example:  Realtime emulation
https://www.tradingview.com/lightweight-charts/

https://jsfiddle.net/TradingView/yozeu6k1/
